test/stresstest/README.md

```diff
@@ -0,0 +1,13 @@
+The stresstest can be invoked by running: ''' python3 stresstest.py '''
+Before starting the stresstest run either: ''' python3 broker.py ''' or ''' python3 server.py '''
```

Comment:
I think you should switch lines 1 and 2 as starting a server is the first step to be done.